buildSrc/src/main/java/Config.kt

```diff
@@ -1,12 +1,13 @@
 import java.math.BigDecimal
 
 object Config {
-    val kotlinVersion = "1.4.32"
+    val kotlinVersion = "1.5.31"
```

Comment:
Hm, can't we even bump it to 1.6 because of this?
![image](https://user-images.githubusercontent.com/4999776/142445915-5aaab015-a9d3-405d-9228-083844fd533d.png)

Reply:
because it's just been released, we cant use the 1.6 features anyway till adoption goes higher, so it's safer to be in a more compatible/adopted version.